### ISSUE TITLE
Design mock ups for version 1.0

### ISSUE THREAD

**Issue by abiduzz420** [VasaviDevelopers/vasavi-appstore#1]
Here is the design for the App screen:

![app-screen](https://i.imgur.com/iHCjoUn.jpg)
Would like to hear your comments cc: @SomeshThakur

**Comment by SomeshThakur** [comment]
Looks Good and I was expecting you to PM mockup. Well, lets proceed with this and add further mock up screens.

**Comment by aravindballa** [comment]
I really suggest you guys use any UI component library to build this. 

https://react-native-training.github.io/react-native-elements/
https://nativebase.io/

This will help you release faster and if it is required you can add custom styling or create few custom components. It also ensures consistency between Android and iOS.

Creating everything from scratch consumes a lot of time.

**Comment by abiduzz420** [comment]
Yeah that's right. I haven't finalized which UI library to use. Most probably, I would be going either with https://shoutem.github.io or React native elements UI.

**Comment by aravindballa** [comment]
Shoutem looks good. It has cool integrations.

Also, don't worry much about the backend. Use something like Firebase for now. We can shift to Mongo later.

**Comment by abiduzz420** [comment]
Yes we're using Firebase. Light weight 😋

**Comment by aravindballa** [comment]
You know what more cool is? Use Google Spreadsheets. People can add new entries via. Google Forms and can even edit them.

Reference: https://developers.google.com/sheets/api/samples/reading
